### ISSUE TITLE
drbd plugin fails

### ISSUE THREAD

**Issue by noci2012** [collectd/collectd#1514]
Jan 20 18:03:12 db001 collectd[17881]: drbd plugin: Unable to parse resource
Jan 20 18:03:22 db001 collectd[17881]: drbd plugin: Unable to parse resource

/proc/drbd contains:
```
version: 8.4.6 (api:1/proto:86-101)
GIT-hash: 833d830e0152d1e457fa7856e71e11248ccf3f70 build by phil@Build64R7, 2015-04-10 05:13:52
 0: cs:Connected ro:Secondary/Primary ds:UpToDate/UpToDate C r-----
    ns:0 nr:397028104 dw:397028104 dr:0 al:0 bm:0 lo:0 pe:0 ua:0 ap:0 ep:1 wo:f oos:0
```
It appearantly chokes on the GIT-hash: line where only <4 columns are expected.... it then complains about `version:' not being a number....

**Comment by mfournier** [comment]
Thanks @noci2012 !

Indeed, this needs to be improved. Would you mind taking a look at, and possibly test #1515 ?

I'm not sure about the `version:` warning you mentioned though. I wasn't able to reproduce this one.

**Comment by noci2012** [comment]
Yes, that is the issue. 
In current kernels a Git tag is shown with lots of blanks in the line, causing the field count to be >4 on the 2nd line.
The actual error is that it tries to identify the drbd id# (which is not a number but the string "version:") in the previous line if >4 columns are seen.

**Comment by mfournier** [comment]
This was released in 5.5.1. Thanks !
